app/survey/utils.tsx

```diff
@@ -74,15 +74,23 @@ const formatAnyMeansTenureChoice = (results: SurveyResults[]): SankeyResults =>
 };
 
 const formatCountryResults = (results: SurveyResults[]): CountryResults => {
```

Comment:
nit: We're making a `countryData` object here and then reformatting to a `CountryResults` - could we save a step and generate `CountryResults` directly?